### DISCUSSION TITLE
[Security Advisory] CVE-2023-2253  in nfd-master and nfd-topology-updater

### DISCUSSION THREAD

**Discussion by sakshisharma84** [kubernetes-sigs/node-feature-discovery#1267] [General]
Hi guys,

We are using the latest version(v0.13.2) of node-feature-discovery and our sysdig scanner shows a vulnerability([CVE-2023-2253](https://nvd.nist.gov/vuln/detail/CVE-2023-2253)) in the following two binaries: nfd-master and nfd-topology-updater (screenshots attached for reference).

Are there any plans to roll out the fixes any time soon?

<img width="1398" alt="Screenshot 2023-07-19 at 2 26 35 PM" src="https://github.com/kubernetes-sigs/node-feature-discovery/assets/8899827/174ad749-34e3-461b-879f-cf7883b1b315">

<img width="1389" alt="Screenshot 2023-07-19 at 2 26 24 PM" src="https://github.com/kubernetes-sigs/node-feature-discovery/assets/8899827/3978f7e3-cbc2-41f6-955e-7dfd513b7ad8">

**Comment by marquiz** [comment]
It's in an indirect dependency brought in by kubernetes. I don't see any reason this particular vulnerability would be relevant to nfd. Moreover, it hasn't been fixed in kubernetes release branches so we cannot update it either.

If you're concerned please report against kubernetes.
